internal/service/credentials/resource_credential_issuer_profile_test.go

```diff
@@ -0,0 +1,199 @@
+package credentials_test
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/pingidentity/terraform-provider-pingone/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingone/internal/verify"
+)
+
+// Note: Issuer Profiles aren't deleted once created [No API]. Deleted only via deletion of the environment.  Placeholder if this changes.
+/*func testAccCheckCredentialIssuerProfileDestroy(s *terraform.State) error {
+	var ctx = context.Background()
+
+	p1Client, err := acctest.TestClient(ctx)
+
+	if err != nil {
+		return err
+	}
+
+	apiClient := p1Client.API.CredentialsAPIClient
+	ctx = context.WithValue(ctx, credentials.ContextServerVariables, map[string]string{
+		"suffix": p1Client.API.Region.URLSuffix,
+	})
+
+	mgmtApiClient := p1Client.API.ManagementAPIClient
+	ctx = context.WithValue(ctx, management.ContextServerVariables, map[string]string{
+		"suffix": p1Client.API.Region.URLSuffix,
+	})
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "pingone_credential_issuer_profile" {
+			continue
+		}
+
+		_, rEnv, err := mgmtApiClient.EnvironmentsApi.ReadOneEnvironment(ctx, rs.Primary.Attributes["environment_id"]).Execute()
+
+		if err != nil {
+
+			if rEnv == nil {
+				return fmt.Errorf("Response object does not exist and no error detected")
+			}
+
+			if rEnv.StatusCode == 404 {
+				continue
+			}
+
+			return err
+		}
+
+		body, r, err := apiClient.CredentialIssuersApi.ReadCredentialIssuerProfile(ctx, rs.Primary.Attributes["environment_id"]).Execute()
+
+		if err != nil {
+
+			if r == nil {
+				return fmt.Errorf("Response object does not exist and no error detected")
+			}
+
+			if r.StatusCode == 404 {
+				continue
+			}
+
+			tflog.Error(ctx, fmt.Sprintf("Error: %v", body))
+			return err
+		}
+
+		return fmt.Errorf("PingOne Credential Issuer Profile %s still exists", rs.Primary.ID)
+	}
+
+	return nil
+}*/
+
+func TestAccCredentialIssuerProfile_Full(t *testing.T) {
+	t.Parallel()
+
+	resourceName := acctest.ResourceNameGen()
+	resourceFullName := fmt.Sprintf("pingone_credential_issuer_profile.%s", resourceName)
+
+	name := acctest.ResourceNameGen()
+	updatedName := acctest.ResourceNameGen()
+
+	initialProfile := resource.TestStep{
+		Config: testAccCredentialIssuerProfile_Full(resourceName, name),
+		Check: resource.ComposeTestCheckFunc(
+			resource.TestMatchResourceAttr(resourceFullName, "id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "environment_id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "application_instance_id", verify.P1ResourceIDRegexp),
+			resource.TestCheckResourceAttrSet(resourceFullName, "created_at"),
```

Comment:
Best to use `verify.RFC3339Regexp` with `TestMatch...`

Reply:
updated / resolved